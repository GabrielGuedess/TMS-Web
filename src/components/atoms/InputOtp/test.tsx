import { render } from '@testing-library/react';

import { InputOTP } from '.';

describe('<InputOTP />', () => {
  it('should render the input OTP correctly', () => {
    const { container } = render(<InputOTP />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
