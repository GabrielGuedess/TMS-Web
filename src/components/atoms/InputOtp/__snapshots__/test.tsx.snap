// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputOTP /> should render the input OTP correctly 1`] = `
<div
  class="relative flex h-min flex-col"
>
  <div
    class="relative"
  >
    <input
      autocomplete="on"
      class="peer w-full rounded-lg border-2 bg-transparent p-3 text-center font-normal text-comet-500 outline-none transition-all dark:border-shark-950 dark:text-white-lilac-50 focus:border-primary-300 dark:focus:border-primary-400"
      id=":r0:"
      placeholder="-"
      type="number"
    />
  </div>
</div>
`;
